apps/server/src/classes/Logger.ts

```diff
@@ -103,8 +103,11 @@ class Logger {
    * @param origin
    * @param text
    */
-  error(origin: string, text: string) {
+  error(origin: string, text: string, notifyElectron = false) {
```

Comment:
suggestion: we dont need to couple this logic to electron, should we instead call it `escalate` as in "you should call the escalate error callback?